.github/workflows/godot-ci.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         run: |
           mkdir -v -p build/windows
           cd $PROJECT_PATH
-          godot -v --export "Windows Desktop" ../build/windows/$EXPORT_NAME.exe
+          godot --headless --verbose --export "Windows Desktop" ../build/windows/$EXPORT_NAME.exe
```

Comment:
I believe `--export` has been deprecated. It is now an explicit `--export-release`, `--export-debug`, or `--export-pack`. There are multiple references to this deprecated flag in this file.

reference: https://docs.godotengine.org/en/stable/tutorials/editor/command_line_tutorial.html